code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -27,6 +27,7 @@
 	var/power = 1
 	//A neutered symptom has no effect, and only affects statistics.
 	var/neutered = FALSE
+	var/stopped = FALSE
```

Comment:
Please add a comment documenting what this var does. Might also want to note what sets it in the comment.

It also seems like if anything else tried to use this var on a disease with `viralincubate` it might cause unexpected behavior, but not sure if there's a better way to do this.